internal/db/schema/migrations/oss/postgres/24/01_session_valid_state.up.sql

```diff
@@ -23,68 +23,68 @@ set termination_reason='canceled' where termination_reason is null;
 
 -- session_valid_state table creation and related constraints
 create table session_valid_state(
-    prior_state text
-        references session_state_enm(name)
-            on delete restrict
-            on update cascade,
-        constraint prior_state_session_state_enm_fkey
-        check (
-                prior_state in ('pending', 'active', 'canceling')
-            ),
-    current_state text
-        references session_state_enm(name)
-            on delete restrict
-            on update cascade,
-        constraint current_state_session_state_enm_fkey
-        check (
-            current_state in ('pending', 'active', 'canceling', 'terminated')
+  prior_state text
+      references session_state_enm(name)
```

Comment:
Should this be indented 2 spaces instead of 4? There are a few places in this PR where we sometimes indent by 4 instead of 2. I can leave comments on all of them if you want.